### DISCUSSION TITLE
学校大内网穿透失败

### DISCUSSION THREAD

**Discussion by Kyokorin0** [opengnb/opengnb#26] [Q&A]
一台笔记本，一台部署在家庭局域网的win_server
在家用lite模式2秒建立连接，移动网络效果相同
在学校10开头的大内网，log显示接受到node发送的ipaddr，却始终建立不了连接，能用数据流量建立连接
请问有什么参数可以增加穿透成功的概率吗

**Comment by gnbdev** [comment]
https://github.com/gnbdev/opengnb/blob/master/docs/gnb_user_manual_cn.md

**Reply by Kyokorin0** [reply]
感谢回复，文档中--upnp和--mutli--socket已经默认打开，因为没有公网ip，暂时没法考虑forwarding节点方案
数据流量访问已经能够满足目前的需求了，大佬的项目很好用，感谢！

**Reply by gnbdev** [reply]
Unified Forwarding

**Reply by Kyokorin0** [reply]
回复，在云服务器上部署了一个gnb节点，通过该节点转发实现了访问位于家庭局域网的服务器，不过还是用到了公网ip

**Reply by gnbdev** [reply]
尽量实现用p2p组网吧，除非学校的网络环境非常复杂；但是务必要注意网络安全，要在学校网络管理员许可的情况下虚拟组网。
